src/args.rs

```diff
@@ -1,12 +1,17 @@
-use std::path::PathBuf;
 use clap::{Parser, ValueEnum};
+use std::path::PathBuf;
 
 #[derive(Parser, Debug)]
 #[command(version, about, long_about = None)]
 pub struct Args {
     #[arg(help = "Path(s) to the file(s) to be opened.", required = true)]
     pub files: Vec<PathBuf>,
 
+    #[arg(long, help = "Select file type", required= true,
+        value_enum,
+        default_value_t = FileType::Csv)]
+    pub filetype: FileType,
```

Comment:
I would suggest changing `filetype` to `format` as the word is usually used with the file structure.